### ISSUE TITLE
[Enhancement]: Hide Tabs When Empty

### ISSUE THREAD

**Issue by chyron8472** [advplyr/audiobookshelf#887]
### Describe the feature/enhancement

I request a toggleable setting for any tabs (Series, Collections, Playlists, et al.) to be hidden from a user if it is empty. I don't use Collections at all, and it would lessen UI clutter if it was hidden.

**Comment by FerriteGiant** [comment]
seconded

**Comment by jesucarr** [comment]
Yes please, I also don't have any Series and I'd like that tab to be hidden when empty

**Comment by ZLoth** [comment]
This should be enabled/disabled at the user level.

**Comment by ZLoth** [comment]
Here is quick mock-up. Just imagine pull-down lists:

![image](https://github.com/user-attachments/assets/95cdce85-c587-4a2d-b8ba-f1a9418dfcf8)
